contracts/staking/StakingMessages.sol

```diff
@@ -84,46 +109,53 @@ library StakingMessages {
         pure
```

Comment:
@minghinmatthewlam pointed out that it's kind of odd for the `pack/unpackRegisterSubnetValidatorMessage` use different message types (one the request to the P-Chain, and the other the response). 

To match the updated ACP-77 spec, maybe we should go with `packRegisterSubnetValidtorMessage` and `unpackSubnetValidatorRegistrationMessage`? Not opposed to adding the request/response nomenclature otherwise either if people think that's more clear.